syntax = "proto3";

service BapidService {
  rpc Ping (PingRequest) returns (PingReply) {}
}

message PingRequest {
  string name = 1;
}

message PingReply {
  string message = 1;
}
